.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[Bug] "
+labels: bug
+assignees: ''
```

Comment:
why exactly is section this here?

Reply:
It's a thing that GitHub adds, these are basically presets for issues that users can select from a list

Reply:
fair enough, just never seen them before so.